visidata/guides/TypesSheet.md

```diff
@@ -10,4 +10,27 @@ Columns usually begin as untyped (`anytype`), but can be set.
 - {help.commands.type-len}
 - {help.commands.type-string}
 
-*Note*: `float` uses the decimal separator; `currency` removes non-numeric characters and parses as float; `date` parses dates into ISO8601 format; `len` formats the cell value to the length of the content.
\ No newline at end of file
+## Type formatting
+
+VisiData pre-set defaults for formatting types:
+
+- `currency` removes non-numeric characters and parses as `float`
+- `date` parses dates into ISO8601 format
+- `float` uses the decimal seperator, keeping two significant digits
+- `len` formats the cell value to the length of the content
```

Comment:
officially it's `vlen`.

Reply:
Should I adjust the docstrings?

visidata/metasheets.py:ColumnsSheet.addCommand('gz#', 'type-len-selected', 'onlySelectedRows.type=vlen', 'set type of selected source columns to len')